__tests__/topics.test.js

```diff
@@ -0,0 +1,29 @@
+const request=require("supertest")
+const seed=require("../db/seeds/seed.js")
+const db=require("../db/connection.js")
+const data=require("../db/data/test-data")
+const app=require("../app.js")
+
+beforeEach(()=>{
+    return seed(data)
+})
+
+afterAll(()=>{
+    return db.end()
+})
+
+describe("/api/topics",()=>{
+    test('GET: 200 - Responds with all key\'s data', () => {
+        return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({body:{topics}})=>{
```

Comment:
There is an assertion that needs to be here 👀 If the response body is empty, the assertions in the .forEach won't run 🫨 So we need an assertion to check that the body has a length property.